envs/linux-pinned.yaml

```diff
@@ -575,6 +575,7 @@ dependencies:
 - zstd=1.5.6
 - pip:
   - chaospy==4.3.17
+  - currencyconverter==0.17.30
```

Comment:
Note please that pinned environments are being automatically updated (e.g. #1321 is an example of an automated PR which fixes this issue). So, luckily there is no need for manual changes